package.json

```diff
@@ -20,7 +20,9 @@
     "build": "gulp build",
     "build:blocks": "gulp build-blocks",
     "build:core": "gulp build-core",
-    "build:debug": "gulp build-core --verbose --strict",
+    "build:compressed":"gulp build-compressed",
+    "build:generators": "gulp build-generators",
+    "build:debug": "gulp build:compressed --verbose --strict",
```

Comment:
this should be ``gulp build-compressed --verbose --strict``

Reply:
done